### PR TITLE
Added new field 'major_label' to editor

### PR DESCRIPTION
**Pull request by daiLlew**
Updated Editor to allow adding/updating major version label.

### CODE REVIEW

File: src/main/java/uk/co/onsdigital/discovery/api/DataResourceAPI.java [hunk 1]
```diff
@@ -78,7 +77,8 @@ public DataResource getDataResourceByID(@PathVariable String dataResourceID) thr
         if (bindingResult.hasErrors()) {
             throw new ValidationException(bindingResult);
         }
-        dataResourceDAO.update(minifyJSON(dataResource));
+        ;
```
Comment (YetAnotherMatt) [inline src/main/java/uk/co/onsdigital/discovery/api/DataResourceAPI.java]: unnecessary semi-colon

File: src/main/java/uk/co/onsdigital/discovery/api/MetadataAPI.java [hunk 2]
```diff
@@ -117,4 +108,23 @@ private DatasetMetadata sanitise(DatasetMetadata metadata) throws UnexpectedErro
...
     }
+
+    private String toCamelCase(String input) throws UnexpectedErrorException {
+        try {
+            if (StringUtils.isEmpty(input)) return input;
+
+            StringBuilder camelCase = new StringBuilder();
+            for (String word : input.trim().split(" ")) {
+
+                String[] letters = word.split("");
```
Comment (YetAnotherMatt) [inline src/main/java/uk/co/onsdigital/discovery/api/MetadataAPI.java]: Could use StringUtils.capitalise(word) instead

**Review by YetAnotherMatt — APPROVED** [review]
A couple of style points, but nothing to stop the merge

File: src/main/java/uk/co/onsdigital/discovery/api/DataResourceAPI.java (hunk 1 above)
Comment (daiLlew) [reply src/main/java/uk/co/onsdigital/discovery/api/DataResourceAPI.java]: removed.

**Pull request merged**